src/main/webapp/profile.js

```diff
@@ -34,14 +34,15 @@ function submitProfileForm(methodType, redirectUrl) {
       qualifiedJobs.push(job.getAttribute('data-enum-name'));
     }
   }
+  okShadow = document.getElementById('ok-shadow-check').checked;
   const personJson = {
     firstName: $('#first-name-field').val(),
     lastName: $('#last-name-field').val(),
     company: $('#company-field').val(),
     job: $('#job-field').val(),
     linkedIn: $('#linkedin-field').val(),
     qualifiedJobs: qualifiedJobs,
-    okShadow: $('#ok-shadow-check').val()
+    okShadow: okShadow
```

Comment:
Nit: just in line this like the rest of the fields
l prefer jQuery bc everything else is jQuery but if it's not worth it don't bother
Reference if you need it: https://stackoverflow.com/questions/901712/how-do-i-check-whether-a-checkbox-is-checked-in-jquery

Reply:
Done